docker-compose.yml

```diff
@@ -34,10 +34,29 @@ services:
     ports:
       - 5050:80
     environment:
-       PGADMIN_DEFAULT_PASSWORD: admin
-       PGADMIN_DEFAULT_EMAIL: admin@qs.com
+      PGADMIN_DEFAULT_PASSWORD: admin
+      PGADMIN_DEFAULT_EMAIL: admin@qs.com
     networks:
-    - quackStack-network
+      - quackStack-network
+  ###################
+  #   Backend      #
+  ###################
+  backend:
+    container_name: quackStack-api
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    ports:
+      - 3000:3000
+    develop:
+      watch:
+        - action: sync
+          path: ./backend/src
+          target: /backend/src
+          ignore:
+            - node_modules
+            - .github/
+            - .git/
```

Comment:
- Container should depend on db being healthy
- Container should health check with `curl localhost:${PORT:-3000}/api/health | grep 'NMP API is healthy and ready!'`